keps/sig-apps/2232-jobs-stopped/README.md

```diff
@@ -193,26 +194,41 @@ spec:
 
 #### Story 2
 
-<!-- An example to motivate stopping active Jobs? -->
+Stopping a running Job allows higher-level controllers to preempt Jobs. While
+such a controller does not exist in-tree currently, Job priority and
+preemption, when implemented and handled, could improve overall completion
+progress through better Job queueing. One can think of the Job preemption to be
+along the same lines as Pod preemption -- when a high-priority Job comes in and
+there aren't enough resources or quota limits left in the cluster to
+accommodate it, one or more low-priority Jobs can be stopped to free up
+resources. After the cluster scales up or after the quota limits are raised,
+the stopped Job can be resumed. The user can also make modifications to the
+Job's mutable fields (e.g. parallelism) in the stopped state before resuming
+it.
+
+Note that stopping a Job for preemption is different from deleting and
+re-creating it because stopping will retain the completion status of the Job.
+The preempting controller can let a Job pick up from where it was stopped
+without losing past completion data.
 
 ### Notes/Constraints/Caveats
 
 While creating a stopped Job is well-understood, stopping an active Job
 requires some careful thought. Assuming that the semantics of stopping a Job
-are similar to that of a Deployment, the controller manager must delete all
-active Pods when a Job is stopped. Pods that successfully completed will still
-count towards the Job's completions even if the Job is stopped.
+are similar to that of pausing a Deployment, the controller manager must delete
+all active Pods when a Job is stopped. Pods that successfully completed will
+still count towards the Job's completions even if the Job is stopped.
 
 Another interesting point of note is that it is important that system
 administrators restricts users from accidentally modifying this field manually
-in a Job managed by a higher-level orchestrator. This can be done with a
-mutating webhook, for example.
+in a Job managed by a higher-level controller. This can be done with a mutating
```

Comment:
validating or mutating webhook.

Also add webhooks to non-goals

Reply:
what do you mean webhooks is not a goal?